src/popover/Popover.js

```diff
@@ -174,43 +172,70 @@ const Popover = React.createClass({
     }
   },
 
-  onPopoverStateChange() {
-    const { onShow, onHide } = this.getPopoverProps();
-    const isAbsolute = this.isAbsolute();
-    if (this.state.isOpen) {
-      onShow();
-      if (isAbsolute) {
+  isStatefull(props) {
```

Comment:
fixed now in #69 with last commit (https://github.com/buildo/react-components/commit/ec77641a80bdfc03a5964ea3a0f0d86470df74ce)